blog/docs/articles/eventing-rabbitmq-announcement.md

```diff
@@ -0,0 +1,62 @@
+# Announcing Eventing RabbitMQ moving to GA
+
+**Author: Gab Satchi, Senior Software Engineer @ VMware**
+
+**Date: 2022-07-26**
+
+We’re excited to announce that [Eventing RabbitMQ](https://github.com/knative-sandbox/eventing-rabbitmq) is now GA (General Availability). With this milestone, the APIs provided will remain stable in future iterations.
+A quick start can be found [here](https://knative.dev/docs/eventing/broker/rabbitmq-broker/).
+
+# Key Features
+
+### Use any RabbitMQ instance
+
+Eventing RabbitMQ works well with RabbitMQ clusters provisioned using the [cluster operator](https://github.com/rabbitmq/cluster-operator) but can also be configured to use RabbitMQ instances provisioned by other means.
+Both broker and source have a `rabbitmqClusterReference` which can either reference a `RabbitMQCluster` created using the cluster operator or a `Secret` containing the credentials to a RabbitMQ instance.
+A full guide can be found [here](https://github.com/knative-sandbox/eventing-rabbitmq/tree/49787466f88b21fe022216a13c8321b334f92dc4/samples/external-cluster).
+
+### FIFO vs Throughput
+
+Eventing RabbitMQ, by default, operates in a first-in-first-out (FIFO) mode for events. However, there can be use cases where ordering isn't required and higher throughput may be preferred.
+Bother broker and source have a `parallelism` annotation (default 1) that can be configured to process more than 1 message in parallel. An example outlining both FIFO and high throughput modes can be found [here](https://github.com/knative-sandbox/eventing-rabbitmq/tree/49787466f88b21fe022216a13c8321b334f92dc4/samples/trigger-customizations).
+
+### Queue Type
+
+Quorum queues are a fault tolerant replicated queue type to be used in multi-node setups. The performance between quorum queues and classic queues are very close and Eventing RabbitMQ has opted to using quorum queues by default.
+The queue type remains configurable at the broker level and any triggers subscribed to a broker will inherit the queue type. Queue type can be configured using the `RabbitMQBrokerConfig` object:
+
+```yaml
+apiVersion: eventing.knative.dev/v1alpha1
+kind: RabbitmqBrokerConfig
+...
+spec:
+  queueType: "classic" #'classic | quorum'
+...
+```
```

Comment:
```suggestion
Quorum queues are a fault tolerant, replicated queue type that are used in multi-node setups. RabbitMQ uses quorum queues by default.

The queue type can be configured for RabbitMQ Brokers, and any Triggers subscribed to a Broker will inherit the queue type. You can configure the queue type for a `RabbitMQBrokerConfig` object as follows:

```yaml
apiVersion: eventing.knative.dev/v1alpha1
kind: RabbitmqBrokerConfig
...
spec:
  queueType: "classic" #'classic | quorum'
...
```
```

Tbh I think this example should be contained in the main docs somewhere instead and linked to from here, rather than including specific YAML configs in a blog